plugins/auth-backend/config.d.ts

```diff
@@ -179,18 +179,6 @@ export interface Config {
         };
       };
       /** @visibility frontend */
-      microsoft?: {
```

Comment:
Shouldn't we deprecate this instead of just removing it? As a user having the old config under auth it still works, right?

Reply:
@jhaals, I've seen both done in the PRs for the other providers. This should continue to work as the new config is still under `auth` as well. But we should maybe be more consistent in our approach

Reply:
In that case I'm happy with this 👍